pkg/kubectl/cmd/util/helpers_test.go

```diff
@@ -179,6 +182,146 @@ func TestMerge(t *testing.T) {
 				},
 			},
 		},
+		{
+			kind: "Pod",
+			obj: &api.Pod{
+				ObjectMeta: api.ObjectMeta{
+					Name: "foo",
+				},
+			},
+			fragment: `{ "apiVersion": "v1" }`,
```

Comment:
As in other test cases in this file, please use testapi.Version() instead of hardcoding v1 apiversion